.woodpecker/.test-build.yml

```
@@ -1,16 +1,15 @@
-pipeline:
-  lint:
+steps:
+  - name: lint
```

This didn't need to be changed, [both versions are supported](https://woodpecker-ci.org/docs/usage/workflow-syntax), but we use the dictionary variant in the other configs already, so I would match that.
```suggestion
  lint:
```